lib/improver/tests/utilities/test_spatial.py

```diff
@@ -225,6 +226,34 @@ def test_projection_transform(self):
         self.assertAlmostEqual(expected_y, result_y)
 
 
+class Test_transform_grid_to_lat_lon(IrisTest):
+    """
+    Test function that transforms the points in the cube
+    into grid of latitudes and longitudes
+
+    """
+
+    def setUp(self):
+        """Set up the cube."""
+        self.cube = set_up_cube(zero_point_indices=((0, 0, 1, 1),),
+                                num_grid_points=2)
+        self.cube.coord(axis='x').points = np.array([-1158000.0, 924000.0])
+        self.cube.coord(axis='y').points = np.array([-1036000.0, 902000.0])
+
```

Comment:
Please add a basic test to assert that the returned arrays are Numpy arrays as I think the existing test will succeed if they are lists instead.

Reply:
Done